tests/app.test.js

```diff
@@ -0,0 +1,28 @@
+import assert from 'assert'
+import { app } from '../src'
+import { CODES } from '../src/constants'
+let operation
+app.Operation.setOutputs(['SUCCESS', 'ERROR'])
+
+describe('app', function () {
+  describe('app.Operation', function () {
+    beforeEach(function () {
+      operation = new app.Operation()
+    })
+    it('it should set outputs correctly', function () {
+      const { SUCCESS, ERROR } = operation.outputs
+      assert.deepStrictEqual(SUCCESS, 'SUCCESS')
+      assert.deepStrictEqual(ERROR, 'ERROR')
+    })
+    it('it should add listeners correctly', function () {
+      operation.on('SUCCESS', function () {})
```

Comment:
Looks like this test is useless

Reply:
this simply checks that you can add an operation function based on an existing output. Just checks that the `.on` method does not throw.